saleor/graphql/order/types.py

```diff
@@ -1158,6 +1158,9 @@ class Meta:
     def resolve_created(root: models.Order, _info):
         return root.created_at
 
+    def resolve_channel(root: models.Order, info):
```

Comment:
Missing `staticmethod` declaration:

```suggestion
    @staticmethod
    def resolve_channel(root: models.Order, info):
```